ios/Exponent/Kernel/ReactAppManager/EXReactAppManager.m

```diff
@@ -123,26 +110,22 @@ - (void)rebuildBridge
   // because Expo.plist is not available in the Expo Go app.
   NSAssert(_appRecord.scopeKey, @"Experience scope key should be nonnull when getting initial properties for root view. This can occur when the manifest JSON, loaded from the server, is missing keys.");
 
-
+#ifndef __cplusplus
```

Comment:
```suggestion
```
maybe we can remove this if because it's now in .m file?

Reply:
No, we can't remove it. Something needs to import `RCTCxxBridgeDelegate` and make `EXReactAppManager` conform to it. We no longer can do this from the `.m` file, but we can in the header file.

Reply:
the #ifndef __cplusplus is inside the EXReactAppManager.m which supposedly no one will import the file? though it's not clear to me still, it has no problem to leave this here.

Reply:
Ah you're right - I messed this up with another `#ifdef` in the header 🙈 I'll remove it!